docker/load_mongodump.sh

```diff
@@ -1,10 +1,72 @@
+#!/bin/bash
+
+# Directory of the script
+SCRIPT_DIR="$(dirname "$0")"
+
+# Path to the configuration file (one level up)
+CONFIG_FILE="$SCRIPT_DIR/../docker-compose-dev.yml"
+
+# Check if the correct number of arguments is provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <mongodump-file>"
+    echo "  <mongodump-file> : The path to the MongoDB dump file to be restored."
+    exit 1
+fi
+
 MONGODUMP_FILE=$1
 
-echo "Copying file to docker container"
-docker cp $MONGODUMP_FILE op-admin-dashboard-db-1:/tmp
+# Print debug information
+echo "Script Directory: $SCRIPT_DIR"
+echo "Configuration File Path: $CONFIG_FILE"
+echo "MongoDump File Path: $MONGODUMP_FILE"
+
+# Check if the provided file exists
+if [ ! -f "$MONGODUMP_FILE" ]; then
+    echo "Error: File '$MONGODUMP_FILE' does not exist."
+    exit 1
+fi
+
+# Check if the configuration file exists
+if [ ! -f "$CONFIG_FILE" ]; then
+    echo "Error: Configuration file '$CONFIG_FILE' does not exist."
+    exit 1
+fi
+
+# Print details about the configuration file
+echo "Configuration file details:"
+ls -l "$CONFIG_FILE"
+
+# Extract database name from the mongodump file
+TEMP_DIR=$(mktemp -d)
+tar -xf "$MONGODUMP_FILE" -C "$TEMP_DIR"
+DB_NAME=$(find "$TEMP_DIR/dump" -mindepth 1 -maxdepth 1 -type d -exec basename {} \;)
+
+if [ -z "$DB_NAME" ]; then
+    echo "Error: Failed to extract database name from mongodump."
+    exit 1
+fi
+
```

Comment:
This works, but something `tar tzvf ... | head -n 1` would be better, since we won't need to extract the fully mongodump locally - the extra space and I/O may matter for large files.

Can you clarify why 

```
DB_NAME=`tar tzvf ... | head -n 1`
```

doesn't work?

Reply:
@shankari For some reason when I did the tar tzvf, it couldn't find the name. I tried with grew, awk, sed, etc but still came out empty handed despite it clearly showing 'open-ca-ebike'. I could try again. Moved this to #122